app/routes/lending/LendableObjectsAdmin.tsx

```diff
@@ -40,31 +78,133 @@ const LendableObjectsAdmin = () => {
+      startTime: moment().subtract({ hours: 2 }),
+      endTime: moment(),
+      message: 'Jeg vil gjerne låne Soundboks til hyttetur:)',
+      approved: false,
+      status: status.APPROVED,
+      lendableObject: {
+        id: 2,
+        title: 'Grill',
+        image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+      },
+    },
+    {
+      id: 4,
+      user: 'Test Testesen',
+      startTime: moment().subtract({ hours: 2 }),
+      endTime: moment(),
+      message: 'Jeg vil gjerne låne Soundboks til hyttetur:)',
+      approved: false,
+      status: status.DENIED,
+      lendableObject: {
+        id: 2,
+        title: 'Grill',
+        image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+      },
+    },
+  ];
+
+  const lendableObjects: Array<ListLendableObject> = [
+    {
+      id: 1,
+      title: 'Grill',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 2,
+      title: 'Soundboks',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 3,
+      title: 'Soundboks2',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 4,
+      title: 'Prinsessekjole',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 5,
+      title: 'Falk',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
   ];
 
+  const [showFetchMore, setShowFetchMore] = useState(true);
+
   return (
     <Content>
       <Helmet title="Utlån" />
-      <NavigationTab title="Utlånsforespørsler" />
+      <NavigationTab 
+        title="Utlånsforespørsler" 
+        back={{
+          label: 'Tilbake',
+          path: '/lending',
+        }}
+      />
       <h2 className={styles.heading}>Ventende utlånsforespørsler</h2>
-      {lendingRequests
-        .filter((request) => !request.approved)
-        .map((request) => (
-          <LendingRequest key={request.id} pending={true} request={request} />
-        ))}
+      <Flex column gap={10}>
+        {lendingRequests
+          .filter((request) => request.status === status.PENDING)
+          .map((request) => (
+            <LendingRequest key={request.id} pending={true} request={request} />
+          ))}
+      </Flex>
+      <br />
       <h2 className={styles.heading}>Godkjente utlånsforespørsler</h2>
-      {lendingRequests
-        .filter((request) => request.approved)
-        .map((request) => (
-          <LendingRequest key={request.id} pending={false} request={request} />
+      <Flex column gap={10}>
+        {lendingRequests
+          .filter((request) => request.status !== status.PENDING)
+          .map((request) => (
+            <LendingRequest key={request.id} pending={false} request={request} />
+          ))}
+          {showFetchMore && (
+            <Button onClick={() => {}}>Last inn mer</Button>
```

Comment:
This doesn't do much

Reply:
Yes, I was just making the gui, while waiting for more functionality to be developed.